spyder/plugins/variableexplorer/widgets/collectionsdelegate.py

```diff
@@ -119,22 +119,16 @@ def createEditor(self, parent, option, index, object_explorer=False):
                 message = _("Spyder is unable to show the {val_type} or object"
                             " you're trying to view because <tt>{module}</tt>"
                             " is not installed. ")
```

Comment:
```suggestion
                message = _("Spyder is unable to show the {val_type} object "
                            "you're trying to view because the "
                            "<tt>{module}</tt> is missing. Please install that "
                            "package in your Spyder environment to fix this "
                            "problem.")
```

* This re-adds the second part of the message that you removed below (i.e. `Please install ...`) because it's the part that tells users how to solve this issue by themselves.
* I reworded the message a bit too because its meaning was not entirely clear.
* Also, please change `val_type = 'dataframe, series'` in line `118` to be `val_type = 'dataframe or series'` (I can't leave a suggestion for that, sorry).

Reply:
I will remove `the` before `<tt>{module}</tt>` from your suggestion: `because numpy is missing` is better than `because the numpy is missing`.